README.md

```diff
@@ -1,5 +1,21 @@
-## Running locally
+# Sentinel Hub openEO backend
+
+## Overview
+
+The Sentinel Hub openEO backend provides access to Sentinel Hub via an openEO interface. 
+
+### Deployments
+
+There are two different deployments (production and test). Although the test version offers slightly more functionality, it is not considered stable and new functions may still change slightly.
+
+The backend can be accessed either directly via existing Sentinel Hub credentials (ClientID + Secret), but is also integrated into the [openEO Platform](https://openeo.cloud/) (EGI Check-in).
```

Comment:
This sentence is a bit strange. Usually it's `either x or y`. Focusing on the beginning of the sentence, the backend itself is accessible without credentials - this sentence as a whole talks about authentication.

I would change this to:
Authentication on the backend can be done either with existing Sentinel Hub credentials (ClientID + Secret) or with openEO Platform credentials (EGI Check-in).

(I left out the integration with openEO Platform for simplicity)

Reply:
Thanks, the way you wrote is even better now.